docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -380,27 +393,72 @@ export const getTagData = (tagName: string, comment?: CommentData) =>
 export const getAllTagData = (tagName: string, comment?: CommentData) =>
   comment?.tags?.filter(tag => tag.tag === tagName);
 
+const getCleanPlatformName = (platform: CommentTagData) => {
+  if (platform.text.includes('ios')) return 'ios';
+  if (platform.text.includes('android')) return 'android';
+  if (platform.text.includes('web')) return 'web';
+  if (platform.text.includes('expo')) return 'expo';
+  return undefined;
+};
+
+const renderPlatformIcon = (platform: CommentTagData) => {
+  const iconProps = { size: iconSize.micro, css: STYLES_PLATFORM_ICON };
+
+  switch (getCleanPlatformName(platform)) {
+    case 'ios':
+      return <AppleIcon color={theme.palette.blue['900']} {...iconProps} />;
+    case 'android':
+      return <AndroidIcon color={theme.palette.green['900']} {...iconProps} />;
+    case 'web':
+      return <AtSignIcon color={theme.palette.orange['900']} {...iconProps} />;
+    case 'expo':
+      return (
+        <ExpoGoLogo
+          width={iconProps.size}
+          height={iconProps.size}
+          color={theme.palette.purple['900']}
+          css={STYLES_PLATFORM_ICON}
+        />
+      );
+    default:
+      return undefined;
+  }
+};
+
 const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
 
 const formatPlatformName = (name: string) => {
   const cleanName = name.toLowerCase().replace('\n', '');
-  return cleanName.includes('ios')
-    ? cleanName.replace('ios', 'iOS')
-    : cleanName.includes('expo')
-    ? cleanName.replace('expo', 'Expo Go')
-    : capitalize(name);
+  if (cleanName.includes('ios')) {
```

Comment:
Thanks for splitting this up into `if`/`else`.

This will change some with the `enum` change above.